src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -271,10 +641,18 @@ public void binaryContentMediaType(final String uri) {
     public void binaryContentNoTypeExternalType(final String uri) {
         final TestInfo info = setupTest("3.9-F-2", "binaryContentNoTypeExternalType",
                 "Fedora servers must use the value of the type attribute in the external content link as the media " +
-                        "type of the external content, if provided. Servers may use the media type obtained when " +
-                        "accessing the external content via the specified scheme (e.g. the Content-Type header for " +
-                        "external content accessed via http).",
+                        "type of the external content, if provided. If there is no type attribute: " +
+                        "Servers may use the media type obtained when accessing the external content via the " +
+                        "specified scheme (e.g. the Content-Type header for external content accessed via http).",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final Headers headers = new Headers(
+                new Header(LINK, externalContentHeader(externalUri, "proxy")),
+                new Header(SLUG, info.getId()));
```

Comment:
Please avoid using the `Slug` header where possible as support for it is not required AFAIK.